src/components/essay/essay.config.yml

```diff
@@ -0,0 +1,38 @@
+title: "Essay"
+status: "alpha"
+collated: true
+context:
+    columns:
+      - stats:
+        - number: "1496"
+          caption: "Labore occaecat excepteur elit"
+      - stats:
+        - link: "#"
+          bgColor: "crimson-dark"
+          number: "39"
+          caption: "Years old is old"
+      - stats:
+        - bgColor: "crimson"
+          number: "38"
+          caption: "Years old is old"
+variants:
+    - name: "images"
+      context:
+          columns:
+            - stats:
+              - image: "https://www.fillmurray.com/400/600"
+                large: true
+            - stats:
+              - link: "#"
+                number: "1496"
+                caption: "Labore occaecat excepteur elit"
+              - link: "#"
+                image: "https://www.fillmurray.com/600/400"
+            - stats:
+              - bgColor: "crimson-dark"
+                number: 39
+                caption: "Years old is old"
+              - link: "#"
+                bgColor: "crimson"
+                number: 38
+                caption: "Years old is old"
```

Comment:
Let's update this placeholder copy to something more generic/random than my stream of consciousness typing from the prototype. Also, 39 in not _that_ old. 😉